.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+main
```

Comment:
I'd add a comment above this to make sure whoever checks it out understands this is the binary file, not main.go